build/dependencies.props

```diff
@@ -5,7 +5,9 @@
     <CoreFxVersion>4.4.0-*</CoreFxVersion>
     <DotnetDebToolVersion>2.0.0-preview2-*</DotnetDebToolVersion>
     <InternalAspNetCoreSdkVersion>2.1.0-*</InternalAspNetCoreSdkVersion>
+    <JsonNetInMSBuildVersion>9.0.1</JsonNetInMSBuildVersion>
     <MoqVersion>4.7.49</MoqVersion>
+    <MsBuildPackageVersions>15.1.1012</MsBuildPackageVersions>
```

Comment:
The current versions we use in our repos is 15.1.1012, see https://github.com/search?utf8=%E2%9C%93&q=org%3Aaspnet+filename%3Adependencies.props+MsBuildPackageVersions&type=Code. Is there a reason why we can use a higher version here or does this need to be updated everywhere?